content/association/statutes.md

```diff
@@ -0,0 +1,79 @@
++++
++++
+
+
+> Note:
+>
+> This page is a draft and is subject to change. The association is not yet registered
+> and we are still considering options on how and where it should be. To help us
+> develop these statutes or the association you can join us in the
+> [Biweekly Nordic RSE meeting](/communities/join/#bi-weekly-meetings).
+> 
+
+
```

Comment:
In Finland, statues must be written in Finnish or Swedish.
```suggestion
In Finland, statues must be written in Finnish or Swedish, but English translation is equivalent will be used by the board in the case of ambiguity.
```

Should it somehow say what the status of the translation is?  I guess legally, there is only Swedish and statues can't say English overrides it, but maybe "in case of ambiguity" doesn't actually override it (since if Swedish was clear, then there isn't ambiguity), but implies that English is the primary working copy.  maybe a professional has to comment, or we leave it out completely???

Reply:
I cannot really comment of whether this is allowed. The Swedish statutes are the statutes, but they maybe could refer to a different document, which would be in English.

In practice, the annual meeting makes ultimate decisions.